sites/public/pages/applications/contact/name.tsx

```diff
@@ -45,6 +50,20 @@ export default () => {
   const emailPresent: string = watch("applicant.emailAddress")
   const noEmail: boolean = watch("applicant.noEmail")
   const clientLoaded = OnClientSide()
+  if (!autofilled && clientLoaded && application.autofilled) setAutofilled(true)
+
+  const AutofillIcon = () => {
```

Comment:
(Non-blocking) I find this name to be a lil confusing, maybe something like DisabledIcon or LockIcon?

Reply:
That makes sense.